tools/tests/centos7/gf-builder

```diff
@@ -3,18 +3,22 @@ FROM centos:7.9.2009
 ENV container docker
 ENV PYTHON /usr/bin/python2
 
-RUN yum install -y systemd epel-release
+#RUN yum install -y --setopt=tsflags=nodocs systemd epel-release && yum clean all
 
-RUN yum install -y \
-        @"Development Tools" diffutils util-linux file \
+RUN yum install -y --disablerepo=extras --setopt=tsflags=nodocs --exclude=kernel-debug-devel --exclude=kernel-devel --exclude=systemtap-devel --exclude=systemtap --exclude=gcc-gfortran --exclude=subversion \
+        @"Development Tools" systemd diffutils util-linux file \
         hostname which less e2fsprogs xfsprogs attr net-tools procps psmisc \
         nfs-utils bind-utils iproute bc vim-common dbench git sysvinit-tools initscripts cronie \
         libselinux-utils perl-Test-Harness yajl rsync lvm2 python3 python-devel python3-prettytable \
         pyxattr gperftools-devel openssh-server openssl openssl-devel \
         libuuid-devel libacl-devel libaio-devel libxml2-devel \
-        userspace-rcu-devel readline-devel && \
+        readline-devel && \
     yum clean all
 
+RUN yum install -y --setopt=tsflags=nodocs --disablerepo=* --enablerepo=extras epel-release && yum clean all
```

Comment:
I wouldn't clean the yum cache after installation. It will make next installation slower and it shouldn't be really needed because `podman build` command uses the `--squash` option, which combines all intermediate layers.

Container images normally also have the `tsflags=nodocs` option predefined, but I'm ok to add it here explicitly.

Reply:
ACK, will remove the cleaning.